sdk/tests/conformance2/textures/misc/tex-unpack-params.html

```diff
@@ -422,85 +442,137 @@
-      alignment: 2, rowLength: 0, imageHeight: 0, skipPixels: 0, skipRows: 0, skipImages: 0 },
+      alignment: 2, rowLength: 0, imageHeight: 0, skipPixels: 0, skipRows: 0, skipImages: 0,
+      validUnpackParams2D: true, validUnpackParamsForSub2D: true,
+      validUnpackParams3D: true, validUnpackParamsForSub3D: true },
     { width: 6, height: 7, depth: 3, xoffset: 2, yoffset: 3, zoffset: 1,
-      alignment: 4, rowLength: 0, imageHeight: 0, skipPixels: 0, skipRows: 0, skipImages: 0 },
+      alignment: 4, rowLength: 0, imageHeight: 0, skipPixels: 0, skipRows: 0, skipImages: 0,
+      validUnpackParams2D: true, validUnpackParamsForSub2D: true,
+      validUnpackParams3D: true, validUnpackParamsForSub3D: true },
     { width: 5, height: 8, depth: 3, xoffset: 2, yoffset: 3, zoffset: 1,
-      alignment: 8, rowLength: 0, imageHeight: 0, skipPixels: 0, skipRows: 0, skipImages: 0 },
+      alignment: 8, rowLength: 0, imageHeight: 0, skipPixels: 0, skipRows: 0, skipImages: 0,
+      validUnpackParams2D: true, validUnpackParamsForSub2D: true,
+      validUnpackParams3D: true, validUnpackParamsForSub3D: true },
 
     // ROW_LENGTH == width
     { width: 10, height: 9, depth: 2, xoffset: 2, yoffset: 3, zoffset: 1,
-      alignment: 4, rowLength: 10, imageHeight: 0, skipPixels: 0, skipRows: 0, skipImages: 0 },
+      alignment: 4, rowLength: 10, imageHeight: 0, skipPixels: 0, skipRows: 0, skipImages: 0,
+      validUnpackParams2D: true, validUnpackParamsForSub2D: true,
+      validUnpackParams3D: true, validUnpackParamsForSub3D: true },
 
     // ROW_LENGTH < width
     { width: 5, height: 7, depth: 3, xoffset: 2, yoffset: 3, zoffset: 1,
-      alignment: 1, rowLength: 4, imageHeight: 0, skipPixels: 0, skipRows: 0, skipImages: 0 },
+      alignment: 1, rowLength: 4, imageHeight: 0, skipPixels: 0, skipRows: 0, skipImages: 0,
+      validUnpackParams2D: false, validUnpackParams3D: false },
     { width: 5, height: 7, depth: 3, xoffset: 2, yoffset: 3, zoffset: 1,
-      alignment: 2, rowLength: 4, imageHeight: 0, skipPixels: 0, skipRows: 0, skipImages: 0 },
+      alignment: 2, rowLength: 4, imageHeight: 0, skipPixels: 0, skipRows: 0, skipImages: 0,
+      validUnpackParams2D: false, validUnpackParams3D: false },
     { width: 5, height: 7, depth: 3, xoffset: 2, yoffset: 3, zoffset: 1,
-      alignment: 4, rowLength: 4, imageHeight: 0, skipPixels: 0, skipRows: 0, skipImages: 0 },
+      alignment: 4, rowLength: 4, imageHeight: 0, skipPixels: 0, skipRows: 0, skipImages: 0,
+      validUnpackParams2D: false, validUnpackParams3D: false },
     { width: 5, height: 7, depth: 3, xoffset: 2, yoffset: 3, zoffset: 1,
-      alignment: 8, rowLength: 4, imageHeight: 0, skipPixels: 0, skipRows: 0, skipImages: 0 },
+      alignment: 8, rowLength: 4, imageHeight: 0, skipPixels: 0, skipRows: 0, skipImages: 0,
+      validUnpackParams2D: false, validUnpackParams3D: false },
 
     // ROW_LENGTH > width
```

Comment:
I thought that ROW_LENGTH > width was supposed to generate an invalid operation. Specifically:
SKIP_PIXELS + width > ROW_LENGTH
SKIP_ROWS + height > IMAGE_HEIGHT

are supposed to generate INVALID_OPERATION.